var/spack/repos/builtin/packages/libsodium/package.py

```diff
@@ -29,14 +29,21 @@ class Libsodium(Package):
     """Sodium is a modern, easy-to-use software library for encryption,
     decryption, signatures, password hashing and more."""
     homepage = "https://download.libsodium.org/doc/"
-    url      = "https://download.libsodium.org/libsodium/releases/libsodium-1.0.3.tar.gz"
```

Comment:
For the record it doesn't matter, but I share your sentiment. The URL is converted into a regular expression when searching for new remote versions, so the numbers are just replaced with `\d`.